backend/src/main/resources/application.yaml

```diff
@@ -7,6 +7,9 @@ spring:
     username: simple_report_app
     password: api123
     url: jdbc:postgresql://localhost:${SR_DB_PORT:5432}/simple_report
+    hikari:
+      maximum-pool-size: 5 # Maximum number of connections to which Hikari should be limited.
+      max-lifetime: 60000 # Maximum lifetime for a connection to be retained in the pool, in milliseconds, once it is closed.
```

Comment:
If the max right now is 30 minutes, I'm wondering if we should bump this to a more conservative threshold first before dropping all the way to 60s. Maybe let's try 10 or 15 minutes first, then drop lower if we don't see issues with that. @jdorothy your thoughts?

Reply:
So my understanding is that these settings are effectively the same, in that a PG connection can be recreated in milliseconds, but I agree that maybe `max-lifetime: 600000` is the way to go here _just in case_ the connection re-creation overhead matters. I think optimizing these settings is going to be a work in progress.

Reply:
I'm more than happy to make that change!